indexer/mailbox/src/message.rs

```diff
@@ -0,0 +1,136 @@
+use sha3::{Digest, Keccak256};
+
+use primitive_types::H256;
+
+use crate::encode::{Decode, DecodeError, Encode};
+
+/// Copied from https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/hyperlane-core/src/types/message.rs
+/// as a workaround due to dependencies of `hyperlane-core` that are not WASM-compatible.
+///
+/// Consider ripping these types out into their own crate.
+
+const HYPERLANE_MESSAGE_PREFIX_LEN: usize = 77;
+
+/// A Stamped message that has been committed at some nonce
```

Comment:
Stamped?

Reply:
Lol this is copied from the monorepo so this must be a very old relic. Will remove